modules/apps/27-interchain-accounts/controller/ibc_middleware.go

```diff
@@ -163,3 +167,22 @@ func (im IBCModule) OnTimeoutPacket(
 
 	return im.app.OnTimeoutPacket(ctx, packet, relayer)
 }
+
+// SendPacket implements the ICS4 Wrapper interface
+func (im IBCMiddleware) SendPacket(
+	ctx sdk.Context,
+	chanCap *capabilitytypes.Capability,
+	packet exported.PacketI,
+) error {
+	panic("SendPacket not supported for ICA controller module")
```

Comment:
```suggestion
	panic("SendPacket not supported for ICA-auth module. Please use SendTx")
```

This would be much clearer imo. Since controller does do SendPacket, it just doesn't do it here